src/Bicep.Cli/Arguments/GenerateParametersFileArguments.cs

```diff
@@ -35,6 +35,32 @@ public GenerateParametersFileArguments(string[] args) : base(Constants.Command.G
                         i++;
                         break;
 
+                    case "--output-format":
+                        if (args.Length == i + 1)
+                        {
+                            throw new CommandLineException($"The --output-format parameter expects an argument");
+                        }
+                        if (OutputFormat is not null)
+                        {
+                            throw new CommandLineException($"The --output-format parameter cannot be specified twice");
+                        }
+                        OutputFormat = args[i + 1];
```

Comment:
If we only expect a fixed set of values for these arguments, should we use an enum rather than a string to model them? See [here](https://github.com/Azure/bicep/blob/e56a4ab5ea34cb3b3baac5ee9c64c4dbfb7bb184/src/Bicep.Cli/Arguments/FormatArguments.cs#L49-L64) for an example of where we do this today.

Reply:
thanks for the suggestion, I just updated my code, now these properties are enum typed 👍